apps/election-manager/src/utils/DownloadableArchive.test.ts

```diff
@@ -11,22 +11,52 @@ test('file prompt fails', async () => {
   const archive = new DownloadableArchive(kiosk)
 
   kiosk.saveAs.mockResolvedValueOnce(undefined)
-  await expect(archive.begin()).rejects.toThrowError(
+  await expect(archive.beginWithDialog()).rejects.toThrowError(
     'could not begin download; no file was chosen'
   )
 })
 
-test('empty zip file', async () => {
+test('direct file save fails', async () => {
+  const kiosk = fakeKiosk()
+  const archive = new DownloadableArchive(kiosk)
+
+  kiosk.saveAs.mockResolvedValueOnce(undefined)
```

Comment:
Direct file save wouldn't use this API, right?

Reply:
ah yeah forgot to update this